zipline/data/us_equity_pricing.py

```diff
@@ -915,7 +915,8 @@ def _write(self, tablename, expected_dtypes, frame):
                 np.array([], dtype=list(expected_dtypes.items())),
             )
         else:
-            if frozenset(frame.columns) != viewkeys(expected_dtypes):
+            if frozenset(frame.columns) != \
+               frozenset(viewkeys(expected_dtypes)):
```

Comment:
Is there still a point in calling `viewkeys` here if we're just going to pass the input to `frozenset` anyway?

Reply:
good point, updated the PR accordingly